src/app/@ansyn/status-bar/components/tree-view/tree-view.component.ts

```diff
@@ -49,17 +50,45 @@ export class TreeViewComponent implements OnInit, OnDestroy {
 	constructor(@Inject(StatusBarConfig) public statusBarConfig: IStatusBarConfig,
 				public store: Store<IStatusBarState>) {
 		this.dataFilters.forEach((f) => {
-			this.dataInputFiltersItems.push(new TreeviewItem(f));
+			const tree = new TreeviewItem(f);
+			this.uncheckAll(tree);
```

Comment:
Do we still need `uncheckAll()` after you've modified `updateInputDataFilterMenu()`?